app/experimenter/experiments/tests/api/v5/test_mutations.py

```diff
@@ -829,6 +887,7 @@ def test_reject_ending_experiment(self):
                     "publishStatus": NimbusExperiment.PublishStatus.IDLE.name,
                     "statusNext": None,
                     "status": NimbusExperiment.Status.LIVE.name,
+                    "populationPercent": "50",
```

Comment:
What is this testing?  Why would we be mutating the population in the middle of a rejection?

Reply:
OH dangarooski. These needed a value for populationPercent when I was checking whether the value had changed in the serializer. Now that we aren't checking that field, these can be removed